scripts/clean-linters.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+set -e
+
+rm -rf pmd-bin*
```

Comment:
Are these directories or files? If the former, could we add a directory separator to make that clear.